utils/pretty/pretty.go

```diff
@@ -209,14 +209,16 @@ func (p *Pretty) Colorize(c *color.Color, msg string) string {
 
 // ColorizeJSON colors keys and values of stringified JSON. On errors the original string is returned.
 // If color is nil then key/value won't be colorize.
-func (p *Pretty) ColorizeJSON(keyColor *color.Color, valueColor *color.Color, data []byte) []byte {
+func (p *Pretty) ColorizeJSON(keyColor *color.Color, valueColor *color.Color, multiline bool, data []byte) []byte {
 	if p.noColor {
 		return data
 	}
 
 	f := prettyjson.NewFormatter()
-	f.Indent = 0
-	f.Newline = ""
+	if multiline == false {
```

Comment:
!multiline

Reply:
Updated

Reply:
It's go way:

```
➜  a git:(dev) ✗ gometalinter .
main.go:7:5:warning: should omit comparison to bool constant, can be simplified to !a (S1002) (megacheck)
```

Also go source code:

```
➜  src gogrep 'if !' |wc -l 
8661
➜  src gogrep 'if.*== false' |wc -l
17
➜  src 
```

Reply:
I don't like the `false == multiline` it's more "conventional but we never do that anywhere in the code. I think it's better to have consistency. Or if you want just change the parameter with `inline` like that you have a `if inline` ;)

Reply:
Ok guys, I give up on this ;)